Tests/scripts/infrastructure_tests/test_collect_tests.py

```diff
@@ -190,40 +192,40 @@ def _test(monkeypatch, case_mocker: CollectTestsMocker, collector_class: Callabl
      ('myXSIAMOnlyPack', 'Whois', 'CoreAlertFields'), None, None),
 
     (MockerCases.C, XSOARNightlyTestCollector,
-     {'myXSOAROnlyTestPlaybook', 'myTestPlaybook', 'Sanity Test - Playbook with Unmockable Whois Integration'},
-     {'bothMarketplacesPack', 'bothMarketplacesPackOnlyXSIAMIntegration', 'myXSOAROnlyPack', 'Whois'}, None, None),
+     ('myXSOAROnlyTestPlaybook', 'myTestPlaybook', 'Sanity Test - Playbook with Unmockable Whois Integration'),
```

Comment:
why change set to tuple?

Reply:
Some of them are set some tuples I just followed the type's intention and it failed to concat set with tuple.